src/Microsoft.DotNet.Darc/src/Darc/Options/DarcOutputType.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.Darc.Options
+{
+    /// <summary>
+    /// Output type that darc should use. Commandline enum parsing is case sensitive
+    /// so we put these as lower case.
+    /// </summary>
+    enum DarcOutputType
+    {
+        yaml,
```

Comment:
```suggestion
        Yaml,
```

Reply:
The command line parsing is case-sensitive, and the rest of our parameters are all lower case. I felt that having to specify "Json" is inconsistent.